### PR TITLE
Add Polish MessagEase layout

### PR DESCRIPTION
**Pull request by Blackweather**
Added a layout in Polish language based on the default layout in MessageEase for Polish Language.
I did not test if this is working, followed the documentation around adding a new layout.

Attaching screenshots of the mentioned MessageEase layout this is trying to mimic.
image
image
image

### CODE REVIEW

**Review by dessalines — CHANGES_REQUESTED** [review]
Check the other PRs for adding keyboards, because you forgot to add this to the keyboard layouts list.

Also run `./gradlew formatKotlin` to pass lint.

File: app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessageEase.kt [hunk 1]
```diff
@@ -0,0 +1,1273 @@
...
+                ),
+                SPACEBAR_KEY_ITEM,
+                RETURN_KEY_ITEM,
+            ),
+        ),
+    )
+
+val KB_PL_MESSAGEEASE: KeyboardDefinition =
+    KeyboardDefinition(
+        title = "polski messageease",
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessageEase.kt:1266]: Messag**E**ase only has one E, so rename this file, this title, and all the vars to only use one E.

**Comment by Blackweather** [comment]
> Check the other PRs for adding keyboards, because you forgot to add this to the keyboard layouts list.
> 
> Also run `./gradlew formatKotlin` to pass lint.

done 
image

File: app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessageEase.kt (hunk 1 above)
Comment (Blackweather) [reply app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessageEase.kt]: corrected the typos

File: app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt [hunk 2]
```diff
@@ -153,6 +154,7 @@ enum class KeyboardLayout(val keyboardDefinition: KeyboardDefinition) {
     ITTypeSplit(KB_IT_TYPESPLIT),
     PTTypeSplit(KB_PT_TYPESPLIT),
     PLTypeSplit(KB_PL_TYPESPLIT),
+    PLMessagease(KB_PL_MESSAGEASE),
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt:157]: Read the big comment at top of this function.

**Review by dessalines — CHANGES_REQUESTED** [review]

**Review by dessalines — APPROVED** [review]
I fixed it up.

**Comment by zworek** [comment]
I was going to make this layout myself.. In fact I did one but then started to mess around with adding dozen of helpers for creating new layouts and than noticed https://github.com/dessalines/thumb-key/pull/700 and now I'm on a really WIP branch 😄 

Anyway thanks for adding it 👍 

> I did not test if this is working, followed the documentation around adding a new layout.

The least I could do is compile and test your code, and there's one bug in it

image

I will add comment in code on how to fix it 😄

File: app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt [hunk 3]
```diff
@@ -0,0 +1,1268 @@
...
+                    center =
+                        KeyC(
+                            display = KeyDisplay.TextDisplay("0"),
+                            action = KeyAction.CommitText("0"),
+                            size = FontSizeVariant.LARGE,
+                            color = ColorVariant.PRIMARY,
+                        ),
+                ),
+                SPACEBAR_KEY_ITEM,
+                RETURN_KEY_ITEM,
```
Comment (zworek) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt:1254]: It brakes spacing with 0 as in my comment


Here is proposed fix:
```diff
diff --git a/app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt b/app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt
index 1ace514..8f64d17 100644
--- a/app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt
+++ b/app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt
@@ -1249,8 +1249,9 @@ val KB_PL_MESSAGEASE_NUMERIC =
                             size = FontSizeVariant.LARGE,
                             color = ColorVariant.PRIMARY,
                         ),
+                    widthMultiplier = 3
                 ),
-                SPACEBAR_KEY_ITEM,
+                SPACEBAR_SKINNY_KEY_ITEM,
                 RETURN_KEY_ITEM,
             ),
         ),
```

Comment (dessalines) [reply app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt:1254]: This just got merged, so fix this in a PR if you would.

Comment (zworek) [reply app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt:1254]: ok, I will create one this week 👍

Comment (dessalines) [reply app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt:1254]: Actually I'm gonna do a release in a min, so I'll correct this now.

Comment (zworek) [reply app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt:1254]: Unfortunately I've found some more small mistakes 😞 

- the minus swipe doesnt produce minus
- it lacks forwardtick
- some special characters have too important color

Comment (dessalines) [reply app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt:1254]: You'll have to fix those in later PRs.

Comment (zworek) [reply app/src/main/java/com/dessalines/thumbkey/keyboards/PLMessagEase.kt:1254]: Addressed in https://github.com/dessalines/thumb-key/pull/777

**Pull request merged**